buildchain/buildchain/image.py

```diff
@@ -178,6 +179,28 @@ def task__image_build() -> Iterator[types.TaskDict]:
         task_dep=['_image_mkdir_root'],
         file_dep=[constants.ROOT/'images'/'salt-master'/'source.key'],
     ),
+    targets.LocalImage(
+        name='keepalived',
+        version='{}-{}'.format(
+            constants.KEEPALIVED_VERSION, constants.KEEPALIVED_BUILD_ID),
```

Comment:
minor: when the opening brace is on its own line, we usually put the closing one on its own as well.

```suggestion
        version='{}-{}'.format(
            constants.KEEPALIVED_VERSION, constants.KEEPALIVED_BUILD_ID
        ),
```

OR

```suggestion
        version='{}-{}'.format(constants.KEEPALIVED_VERSION,
                               constants.KEEPALIVED_BUILD_ID),
```

Ditto for `VERSION` in `build_args` (btw, we may want to factor that in a "local" global in this file.

Reply:
Then `pylint` or whatever should check for it.

Reply:
We could use `black` for formatting, and enforce it in the CI.